docs/docs-components/data/components.js

```diff
@@ -2080,6 +2081,36 @@ const componentData: $ReadOnlyArray<ComponentData> = [
       },
     },
   },
+  {
+    id: 'TableOfContents',
+    platform: {
+      web: {
+        name: 'TableOfContents',
+        visual: {
+          svg: <TableOfContents />,
+        },
+        alias: ['Table of Contents', 'Page Navigation', 'Page Navigator', 'Secondary Nav'],
+        description:
+          'TableOfContents is used to navigate to anchors on a page. It also serves as an outline of a page’s content.',
+        category: ['Navigation'],
+        status: {
+          accessible: {
+            a11yComprehension: 'ready',
```

Comment:
can you add a video in the description of the PR where you test the accessibility of it?

Reply:
I actually copied these values from the coda doc. What are the steps for testing these? Or do we just do basic manual testing?

Reply:
I think in this case we need to check

- Landmarks in the rotor and proper access to TOC when pressing enter
- correct navigation using tab key
- proper communication of list elements

For basic learning how to test this, I recommend reading my book "The Power Of Healing" haha check my training:
https://w.pinadmin.com/display/EPD/Web+Accessibility+Workshop.+Beginner+to+Advanced%3A+How+to+make+Pinterest+accessible+and+enjoyable